### PR TITLE
bump box/spout version to 2.7.2

### PR DESCRIPTION
**Pull request by jmleroux**
This PR aims to use last [2.7.2 stable version of box/spout](https://github.com/box/spout/tree/v2.7.2)
It fixes some issues with blank lines for example. Usefull for the [ExcelInitBundle](https://github.com/akeneo/ExcelInitBundle)

### CODE REVIEW

**Comment by aRn0D** [comment]
@jmleroux What is the status this PR ?

**Comment by jmleroux** [comment]
@aRn0D I had no time to run perfbench, so it's kind of suspended.

**Comment by a2xchip** [comment]
:D :D :D What exactly to review? Build status?

File: CHANGELOG-1.7.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 - PIM-6118: Improve attribute add select to avoid performance impact
 - GITHUB-5716: Redo family mass edit form using backbonejs architecture and internal REST API
 
+## Technical improvements
+- GITHUB-5285: Update to spout 2.7.1
```
Comment (a2xchip) [inline CHANGELOG-1.7.md]: upgrade?

Comment (a2xchip) [inline CHANGELOG-1.7.md]: No empty line after `## Technical improvements`

File: CHANGELOG-1.8.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 - TIP-725: Generalization of the refactoring made in the TIP-724 for all screen containing a simple grid
 - TIP-734: Menu and index page is now using the new PEF architecture
 - GITHUB-6174: Show a loading mask during the file upload in the import jobs
+- GITHUB-5285: Upgrade to spout 2.7.2
```
Comment (a2xchip) [inline CHANGELOG-1.8.md]: :+1:

**Comment by aRn0D** [comment]
I close this PR because it was done in https://github.com/akeneo/pim-community-dev/commit/35053fb1d879850427241a2d1f8884153b464ab4

**Comment by a2xchip** [comment]
:+1:

**Pull request closed without merging**